rules-tests/Php83/Rector/ClassMethod/AddOverrideAttributeToOverriddenMethodsRector/Fixture/apply_attribute_to_override_method_from_trait.php.inc

```diff
@@ -30,10 +26,6 @@ class ApplyAttributeToOverrideMethodFromTrait
     use ExampleFromTrait;
 
     #[\Override]
-    public function foo()
-    {
-    }
-
```

Comment:
the foo and bar override from trait usage in on purpose check, as foo() is override empty, while bar() is not, which cause error, see https://3v4l.org/J50fsl

Reply:
here the proof https://3v4l.org/O2TdJ, I will add check back for it.

Reply:
I create a patch PR for it:

- https://github.com/rectorphp/rector-src/pull/6471